fleet_vehicle_inspection/models/fleet_vehicle_inspection.py

```diff
@@ -13,24 +13,24 @@ class FleetVehicleInspection(models.Model):
 
     READONLY_STATES = {
         "confirmed": [("readonly", True)],
-        "cancel": [("readonly", True)],
+        "canceled": [("readonly", True)],
     }
 
     name = fields.Char(
         "Reference", required=True, index=True, copy=False, default="New"
     )
 
     state = fields.Selection(
-        [("draft", "Draft"), ("confirmed", "Confirmed"), ("cancel", "Cancelled")],
+        [("draft", "Draft"), ("confirmed", "Confirmed"), ("canceled", "Canceled")],
```

Comment:
If you change this, you need to add a migration script to fix databases coming from previous versions.

But since the change doesn't add much value, I would simply avoid doing it. It's up to you :)

Reply:
Ok.. I'm going back to how it was. I don't think it's worth the effort to create a migration script 😄

Reply:
@ivantodorovich done. Thanks :)